__tests__/commands/changelogs/index.test.ts

```diff
@@ -41,8 +43,8 @@ describe('rdme changelogs', () => {
   });
 
   describe('existing changelogs', () => {
-    let simpleDoc;
-    let anotherDoc;
+    let simpleDoc: PageObject;
```

Comment:
It flagged that these didn't have an explicit type. I don't know where to put `page.types.js` but I didn't want to copy-paste this `PageObject` interface in a handful of files.